java/spacewalk-java.spec

```diff
@@ -664,6 +667,11 @@ echo "#### SYMLINKS END ####"
 %service_add_pre taskomatic.service
 %endif
 
+%post
+%if 0%{?rhel}
+systemctl start rngd
```

Comment:
While rng-tools is now only "Recommended": How does this behave if rng-tools is not installed?

Reply:
good point, forgot about that line

Reply:
Fine with me. :+1: